README.md

```diff
@@ -30,23 +30,92 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
```

Comment:
Needs a diagram explaining the difference between a local spawn and a remote spawn and why both exist

Reply:
done.